apps/fxc-front/src/app/components/2d/path-element.ts

```diff
@@ -66,13 +66,17 @@ export class PathElement extends connect(store)(LitElement) {
   private faiSectors?: FaiSectors;
   private plannerElement?: PlannerElement;
   private scoringRequestId = 0;
-  private scoringWorker?: Worker;
+  private scorer?: Scorer = new Scorer(
```

Comment:
Can this be undefined at some point?
If not you should remove the `?`

Reply:
yes it becomes undefined in `destroy` method